be/src/formats/csv/csv_reader.cpp

```diff
@@ -325,6 +328,16 @@ Status CSVReader::more_rows() {
             }
 
         case ORDINARY:
+            //
+            if (UNLIKELY(_parse_options.trim_space)) {
```

Comment:
It's not efficient. You can find the last space in one loop if there are multiple spaces.

Reply:
In fact, we just only need target the first whitespace.